pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -107,6 +126,10 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 			continue
 		}
 		job := jobs.Pop().(*api.JobInfo)
+		if util.Reservation.TargetJob != nil && job.UID == util.Reservation.TargetJob.UID {
```

Comment:
l think this judgment can be abstracted into a function, it is used in many places